custom.d.ts

```diff
@@ -2,8 +2,10 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 declare module '*.svg' {
   import { FunctionComponent, SVGProps } from 'react'
-  const Component: FunctionComponent<SVGProps<SVGSVGElement>>
-  export default Component
+  export const ReactComponent: FunctionComponent<SVGProps<SVGSVGElement>>
+
+  const url: string
+  export default url
```

Comment:
I like having the `url` as the default export, however the rest of JS codebase exports the react component as the default doesn't it?